master/lustrefactory.py

```diff
@@ -56,6 +62,65 @@ def buildzfsCommand(props):
     return args
 
 @util.renderer
+def configureCmd(props):
+    args = ["./configure"]
+    style = props.getProperty('buildstyle')
+
+    if style == "deb" or style == "rpm":
```

Comment:
This kind of thing all seems much more difficult to follow than just having different commands for different platforms.